src/components/NewMovie/NewMovie.tsx

```diff
@@ -1,45 +1,95 @@
 import { useState } from 'react';
 import { TextField } from '../TextField';
+import { Movie } from '../../types/Movie';
 
-export const NewMovie = () => {
-  // Increase the count after successful form submission
-  // to reset touched status of all the `Field`s
-  const [count] = useState(0);
+interface Props {
+  onAdd: (movie: Movie) => void;
+}
+
+export const NewMovie: React.FC<Props> = ({ onAdd }) => {
+  const [count, setCount] = useState(0);
+  const [movie, setMovie] = useState({
+    title: '',
+    description: '',
+    imgUrl: '',
+    imdbUrl: '',
+    imdbId: '',
+  });
+
+  const addNewMovie = (key: string, value: string) => {
+    setMovie({
+      ...movie,
+      [key]: value,
+    });
+  };
+
+  const handleSubmit = (event: { preventDefault: () => void; }) => {
```

Comment:
just get type for event
```suggestion
  const handleSubmit = (event: ...) => {
```